turbo/execution/eth1/forkchoice.go

```diff
@@ -519,6 +520,10 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 			}
 			gasUsedMgas := float64(fcuHeader.GasUsed) / 1e6
 			mgasPerSec := gasUsedMgas / totalTime.Seconds()
+			metrics.GetOrCreateCounter("chaintip_mgas_per_sec").Add(mgasPerSec)
+			metrics.GetOrCreateCounter("chaintip_flush_latency_ms").Add(float64(blockTimings[engine_helpers.BlockTimingsFlushExtendingFork].Milliseconds()))
+			metrics.GetOrCreateCounter("chaintip_validation_latency_ms").Add(float64(blockTimings[engine_helpers.BlockTimingsValidationIndex].Milliseconds()))
```

Comment:
@awskii you had talk about usage `Gauge` instead of `Counter`. Write what need to do one more time here pls. Or `Summary`?

Reply:
Gauge should be used for ? `mgas/sec` actually

Reply:
Use Counter or Summary?